### DISCUSSION TITLE
Any way to set glarePosition to left and right?

### DISCUSSION THREAD

**Discussion by Andyy58** [mkosir/react-parallax-tilt#74] [Ideas]
I'm looking for a way to set the glarePosition to left and right at the same time, but not top or bottom, would anyone know anything that might work?

**Comment by mkosir** [comment]
Hi @Andyy58, thanks for the suggestion, it makes sense to be able to define multiple glare positions!

I think the best approach, not causing  breaking change to current library API, would be to mark current prop `glarePosition` as deprecated and introduce new one `glarePositions` where we would be able to pass multiple positions or `all`.
Prop type signature might look something as:
```tsx
type GlarePosition = ReadonlyArray<'top' | 'right' | 'bottom' | 'left'> | 'all';

const multipleGlarePosition: GlarePosition = ['top', 'right'];
const allGlarePosition: GlarePosition = 'all'; // 360° glare
```
